server/src/server.ts

```diff
@@ -58,7 +57,7 @@ connection.onInitialize(async (params: InitializeParams) => {
       connection.console.error('Cannot install prisma-fmt: ' + err)
     }
   }
-
+  
```

Comment:
are you using prettier? weird to see this spacing

Reply:
I am using prettier, don't know why this happened